package.json

```diff
@@ -70,8 +70,20 @@
     "testRegex": "(\\.|/)(tests)\\.js$",
     "testEnvironment": "node",
     "setupFilesAfterEnv": [
-      "<rootDir>/tests/setupTests.js"
-    ]
+      "<rootDir>/tests/setup-tests.js"
+    ],
+    "testRunner": "jest-circus/runner"
+  },
+  "nyc": {
+    "description": "test coverage",
+    "exclude": [
+      "**/*.test.js",
+      "tests/**"
+    ],
+    "reporter": [
+      "lcov"
+    ],
+    "report-dir": "./coverage"
```

Comment:
Yes, actually I was also working on the `nyc` migration while you were doing the same in your PR. I added the config in `package.json` while you were using CLI args. In that case I'll just remove this config here since I'm not a huge fan of polluted `package.json` files.